WPMedia/fusion-news-theme-blocks
# PEN-1689: Regression on Results List Block styling

**If you have not filled out the checklist below, the pr is not ready for review.**

## Description
Fix for Results List Block regression that was causing images to take up 50% width

## Jira Ticket
- [PEN-1689](https://arcpublishing.atlassian.net/browse/PEN-1689)

## Acceptance Criteria
Should match Zeplin designs https://app.zeplin.io/project/5ea04773df4e1b7ae81167f8/screen/5ea04bb4401a867be62a2743

## Test Steps
1. In `Fusion-News-Theme` repo, checkout `master` & `git pull`
2. After checking out this feature branch in `fusion-news-theme-blocks`, run `rm -rf ./node_modules && npx lerna clean && npm i && npx lerna bootstrap && cd blocks/header-nav-chain-block && npm i && cd ../..`
3. Run `npx fusion start -f -l @wpmedia/results-list-block,@wpmedia/shared-styles` in `Fusion-News-Theme`
4. Open a test page in PB editor and add a Results List Block to the page. Configure it to use `story-feed-query` for content with a query of `type:story`.
5. Stage, publish, open in separate tab
6. You should see that images on desktop breakpoint now take up 35% of the list item width instead of 50%

## Effect Of Changes
### Before
Image on Results List Block list items were taking up 50% width (see https://corecomponents.arcpublishing.com/pf/tags/beaches/?_website=dagen for an example)

### After
Image on Result List Block list items are now taking up only 35% of the item width

## Dependencies or Side Effects
_Examples of dependencies or side effects are:_
- Additional settings that will be required in the blocks.json
- Changes to the custom fields which will require users to reconfigure features
- Update to css framework or SDK
- Dependency on another PR that needs to be merged first

## Author Checklist
_The author of the PR should fill out the following sections to ensure this PR is ready for review._
please explain why (so that we can fix it whenever it gets refactored).

## Reviewer Checklist 
_The reviewer of the PR should copy-paste this template into the review comments on review._
